lib/common/database.dart

```diff
@@ -199,77 +200,62 @@ class DatabaseService {
     }
   }
 
-  Future<Task> getTask(String taskID) async {
-    try {
-      var taskDocument = await users.doc(uid).collection('tasks').doc(taskID).get();
-      return Task.requireFields(
-          name: taskDocument['name'],
-          id: taskID,
-          description: taskDocument['description'],
-          completed: taskDocument['completed'],
-          color: taskDocument['hex color'],
-          location: taskDocument['location'],
-          tags: taskDocument['tags'],
-          recurrenceRules: taskDocument['recurrence rules'],
-          timeStart: taskDocument['start date'],
-          timeDue: taskDocument['due date'],
-          timeCurrent: taskDocument['current date'],
-          timeCreated: taskDocument['date created'],
-          timeModified: taskDocument['date modified']);
-    } catch (e) {
-      print("Get Failed");
-      return Task();
-    }
-  }
-
```

Comment:
old code, not needed and not practical

Reply:
is there a reason why we're keeping setTask then?

Reply:
setTask is still useful for writing a task into database or updating a task's entry in the database

Reply:
personally I think it can be fairly useful for unit tests

Reply:
That's true, I'll add it back in